domains/chip.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Krchip's develop site",
+  "repo": "https://github.com/Krchip/Krchip.github.io",
+  "owner": {
+    "username": "Krchip",
+    "email": "elite7744@gmail.com"
+  },
+  "record": {
+    "URL": "Krchip.github.io"
```

If you wanted  a cname record (subdomain),
```suggestion
    "CNAME": "krchip.github.io"
```

If you wanted a URL redirection,
```suggestion
    "URL": "https://krchip.github.io"
```